monkey/infection_monkey/monkey.py

```diff
@@ -179,9 +181,11 @@ def start(self):
                 for exploiter in [exploiter(machine) for exploiter in self._exploiters]:
                     if self.try_exploiting(machine, exploiter):
                         host_exploited = True
+                        VictimHostTelem('T1210', ScanStatus.USED.value, machine=machine).send()
```

Comment:
Do we need to have this MITRE information outside somewhere? Because we will need a mapping between T1210 and descriptive text somewhere.

Reply:
I don't know. We don't need description for monkey. On island each technique has its description in schema (we add that description to the tooltip in UI). When generating report I could simply pull related description from DB. 
From developer perspective I don't know how to make life easier than having to type the technique id :/

Reply:
I agree that we don't need the description in the Monkey. 
I want to think of how we we will hold this data later on. Will be just have text for all possible techniques or every addition will add to the schema?